### DISCUSSION TITLE
Tabs Section

### DISCUSSION THREAD

**Discussion by joshxyzhimself** [mantinedev/mantine#6021] [Feature requests]
Something to group multiple tabes, like that "Set up", "Stay secure", "Get organized" in fastmail.com.

![image](https://github.com/mantinedev/mantine/assets/60829711/0a0abe8b-9e0b-4630-9646-dfb62de7b467)

**Comment by rtivital** [comment]
https://mantine.dev/core/nav-link/

**Comment by joshxyzhimself** [comment]
That makes sense. Literally laughing right now because I never thought of that, haha. Thank you sir!
